src/main/java/io/vertx/core/impl/BlockedThreadChecker.java

```diff
@@ -45,10 +45,10 @@ public void run() {
           long now = System.nanoTime();
           for (VertxThread thread : threads.keySet()) {
             long execStart = thread.startTime();
-            long dur = now - execStart;
+            long dur = (now - execStart) / 1000000;
```

Comment:
1. please explain why do the ns->ms conversion only on print and not earlier?
2. my commit to `Vertx.java` in this PR reverts the javadoc to declare `timeLimit` in ms - which I think makes more sense when comparing to other timing APIs in Java (see the relevant commit comment), so then the change in line 48 causes `dur` to be in ms so it can be adequately compared to `timeLimit` (the original bug #1812 was about this comparison being wrong, and as I stated in the comments there, I believe that doing this comparison in ms makes more sense).
3. ah, I see. I wasn't aware of that. That makes this whole discussion moot, don't you think? I'll close this PR.